client/src/router/index.js

```diff
@@ -58,23 +58,17 @@ const routes = [
   {
     path: '/about',
     name: 'About',
-    // route level code-splitting
-    // this generates a separate chunk (about.[hash].js) for this route
-    // which is lazy-loaded when the route is visited.
-    component: () => import(/* webpackChunkName: "about" */ '../views/About.vue'),
```

Comment:
Probably want to revert these (and maybe name the chunks in accordance with the routes while you're at it)

Reply:
This branch doesn't have anything to do with routes or naming the chunks so it will just be reverted.